spec/features/edit_check_in_spec.rb

```diff
@@ -2,39 +2,59 @@
 
 feature "Edit check in" do
   include_context "account login"
+  def activity_div(activity)
+    find("#activity-#{activity.id}")
+  end
+
+  def select_quantity(activity, quantity)
+    activity_div(activity)
+      .select quantity, from: "How many hours did you work?"
+  end
+
+  def select_quality(activity, quality)
+    activity_div(activity)
+      .select quality, from: "How'd it go?"
+  end
+
+  def fill_in_notes(activity, content)
+    activity_div(activity)
+      .fill_in("Notes", with: content)
+  end
```

Comment:
I was actually going to add all of this to my tests, I realized I might need these to fill in the check in.